apps/condo/domains/ticket/utils/helpers.ts

```diff
@@ -621,8 +621,14 @@ export function getHumanizeDeadlineDateDifference (ticket: ITicketUIState) {
     const deadlineStopPoint = getDeadlineStopPoint(ticket)
 
     const diff = deadline.diff(deadlineStopPoint, 'day')
-    const moreThanDayDiff = dayjs.duration(diff, 'days').humanize()
     const overdueDiff = dayjs.duration(diff, 'days').subtract(1, 'day').humanize()
+    const moreThanDayDiff = dayjs.duration(diff, 'days').humanize()
+
+    if (diff === 1) {
+        const dividedHumanizedDiff = moreThanDayDiff.split(' ')
+        const customOneDayDiff = `${diff} ${dividedHumanizedDiff[dividedHumanizedDiff.length - 1]}`
```

Comment:
it looks like a hack. Are you sure that the `moreThanDayDiff` do not change their format?

Reply:
`moreThanDayDiff` is a string returned by the `humanized` method.
When the `en` locale is used, `moreThanDayDiff` consists of two words.
When the `ru` locale is used, 
- in case `diff === 1` `moreThanDayDiff` consists of one word; 
- in case `diff > 1` `moreThanDayDiff` consists of two words.